### ISSUE TITLE
Expanding underlining in normalized version

### ISSUE THREAD

**Issue by aewingate** [IUBLibTech/newton_chymistry#102]
In ALCH00049 (Keynes 60), Jim discovered a section at the bottom of 2r and the top of 2v where Newton has used a shorthand version of underlining to indicate that the passage was in italics in the original text. Newton underlines the first word in the passage (Quaedam), the last word of the passage (induratus), and the first syllable of each word that starts a new line. 

This renders as it should in the diplomatic version
![image](https://user-images.githubusercontent.com/72820848/109708893-f5c45780-7b69-11eb-92aa-4f1894ccecd4.png)
but we need the underlining to stretch all the way across in the normalized version. Currently the normalized doesn't even match the diplomatic since some of the <hi rend="u"> elements have to be inside the `<orig>` of `<orig>` and `<reg>`.
Normalized Carbon site
![image](https://user-images.githubusercontent.com/72820848/109709578-ae8a9680-7b6a-11eb-9a20-b0a1a63ca637.png)
Current TEI code
![image](https://user-images.githubusercontent.com/72820848/109709100-299f7d00-7b6a-11eb-9c8d-68d793206bba.png)

@mdalmau has suggested encoding with TEI `<span>` and then using that as the element to attach any CSS or XSLT to for the normalized version. 

I am not sure if this is an issue that comes up in other manuscripts, but since Bill, Jim, and I are going through the manuscripts so granularly, I think we'll catch any other instances.

**Comment by aewingate** [comment]
So doing something like this then?
`	<choice>
		<orig><hi rend="u">sunt</hi>
			<choice><orig>quæ</orig><reg>quae</reg></choice> indurant fixant &amp; congelant &amp;
					<choice><orig>quædam</orig><reg>quaedam</reg></choice> indurantur</orig>
		<reg>
			<hi rend="u"><hi rend="u">sunt</hi>
				<choice><orig>quæ</orig><reg>quae</reg></choice> indurant fixant &amp; congelant &amp;
						<choice><orig>quædam</orig><reg>quaedam</reg></choice> indurantur</hi>
		</reg>
	</choice>`

**Comment by tubesoft** [comment]
From the perspective of rendering algorithm (i.e. not from that of TEI guideline), I suppose that nesting `<choice>` does not make sense much. In other words, `<reg>` inside `<orig>` or `<orig>` inside `<reg>` will probably never be read. If TEI guideline allows, you could put the entire passage that is suppose to be rendered as diplomatic version inside `<orig>`, and the passage for the normalized version inside `<reg>`.

**Comment by tubesoft** [comment]
@aewingate  For your information, my suggestion would go like this:

`<choice> <orig><hi rend="u">sunt quæ indurant fixant &amp; congelant &amp; quædam indurantur</hi></orig> <reg><hi rend="u"><hi rend="u">sunt</hi> quae indurant fixant &amp; congelant &amp; quaedam indurantur</hi></reg> </choice> `

**Comment by aewingate** [comment]
@tubesoft I talked to John Walsh this afternoon, and he came up with this solution for us instead of the duplicating that both of us were doing. 
`<hi rend="u">sunt</hi><hi rend="norm_u"><choice><orig>quæ</orig><reg>quae</reg></choice> indurant fixant &amp; congelant &amp; <choice><orig>quædam</orig><reg>quaedam</reg></choice> indurantur</hi>                       <hi rend="u">fix</hi><hi rend="norm_u">antur et congelantur: Et sic est duplex consideratio in arte</hi><lb/>`

Basically, the part that always needs to be underlined whether it's in the diplomatic or the normalized will have `<hi rend="u">` surrounding it, but then the part which is only underlined in the normalized version will have `<hi rend="norm_u">` surrounding it, which I believe means that there will need to be a change to the XSLT or CSS to match the "norm_u". 
![image](https://user-images.githubusercontent.com/72820848/110871857-2bb9b800-829d-11eb-8989-e37aa559253f.png)

**Comment by tubesoft** [comment]
Yeah, that looks nice! I can modify the XSLT and CSS code so that it can work. However, be careful about making space "sunt" and "quae", because currently the code would be rendered as "suntquae". If you add a space  `</hi>` and `<hi rend="norm_u">` the space is not going to be underlined. Maybe we could add it between `<hi rend="norm_u">` and `<choice>`.

**Comment by aewingate** [comment]
Cool! I will encode ALCH00049 now and put that up into Xubmit. And I definitely see what you mean about the spacing, and I think adding it between `<hi rend="norm_u">` and `<choice>` makes sense.

**Comment by aewingate** [comment]
Alright, I just re-encoded the section and put it in Xubmit. There's one section that was a little tricky because the line break combined with the underlining split a word ("congelatum") into three sections. I think how I've done it should work since "con" doesn't need to be underlined in the diplomatic, and in the regularized it's all one word on the next line and I can wrap the text inside the `<reg>` with `<hi rend="norm_u">`. 
Just wanted to give a heads up in case it acts weird!
![image](https://user-images.githubusercontent.com/72820848/111003361-fd97af00-8354-11eb-98db-64c0f287ebfa.png)

**Comment by tubesoft** [comment]
Thank you, @aewingate ! I should have talked to you before, but I have one small question. the value of the attribute `norm_u` might be better to change. Specifically saying, when I went through other TEI code, I don't think that there are any attribute values that has underscore, except for referring a file whose name contains underscores. Connecting words by hyphen seems more common. This is pretty much the matter of project's coding convention. Judging from this, I would suggest `norm-u` instead of `norm_u`. Maybe you might want to talk to @mdalmau about how to name the attribute value.

**Comment by aewingate** [comment]
I talked to John Walsh, and he says that it's arbitrary, since we've tended to go for hyphens `norm-u` sounds good. I'm going to go make that change now!

**Comment by tubesoft** [comment]
Fixed at #104

**Comment by wehooper** [comment]
The rendering works as prescribed in both the diplomatic and normalized. Nice solution.
